src/Api/requestAsset.js

```diff
@@ -1,10 +1,14 @@
 import axios from 'axios';
+import { constructRequest } from '../State/helpers';
 
 export const getEmployees = args => {
-  const url = `https://e6b8r4dht6.execute-api.us-west-2.amazonaws.com/dev/users`;
-  const fetchBody = {
-    function: 'get_users',
-    params: {}
-  };
-  return axios.post(url, fetchBody).then(response => response.data.users);
+  const url = `https://j8nhpla4d3.execute-api.us-west-2.amazonaws.com/dev`;
```

Comment:
Maybe throw a `TODO` here and on any other URLs you don't have it - I saw a comment about importing a base url constant, so it'd be cool to track everywhere we want to make that change